lib/OpenQA/Script/CloneJob.pm

```diff
@@ -241,10 +241,37 @@ sub clone_jobs ($jobid, $options) {
     handle_tx($tx, $url_handler, $options, $jobs) if $tx;
 }
 
+sub detect_maintaince_update($job) {
+    my %settings = %{$job->{settings}};
+    my %test_template = ('BASE_TEST_TEMPLATE' => 'BASE_TEST_ISSUES', 'DESKTOP_TEST_TEMPLATE' => 'DESKTOP_TEST_ISSUES', 'SDK_TEST_TEMPLATE' => 'SDK_TEST_ISSUES', 'WE_TEST_TEMPLATE' => 'WE_TEST_ISSUES');
+    my @list_url ;
+    for ( keys %test_template) {
+       next  unless ($settings{$_} or  $settings{$test_template{$_}});
+       my $test_template = $settings{$_};
+       my $ua = LWP::UserAgent->new;
+       $ua->timeout(10);
+       foreach my $TEST_ISSUE (split (/,/ ,  $settings{$test_template{$_}})) {
+            my $maintainance_url = $test_template;
+            $maintainance_url =~ s/\@INCIDENTNR\@/$TEST_ISSUE/ ;
+            my $response = $ua->get($maintainance_url);
```

Comment:
So this tries to find the URL for the incident and aborts if that is absent. I wonder if we can make this more generic. A simple URL check could be done that's not as specific about incidents.

Reply:
the clone script has already asset management :) this would apply only to specific issues